xunit-autolabeler-v2/ast_parser/lib/constants.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+IGNORED_METHOD_NAMES = [
+    'run_command',
+    'parse_command_line_args',
+    'main'
+]
+
+REGION_TAG_GREP_ARGS = \
+    'grep -hr START .'.split(' ')
+REGION_TAG_GREP_ARGS += ['--include=*.py', '--exclude=*/lib/*']
+REGION_TAG_GREP_ARGS += ['--include=*.js', '--exclude=*/node_modules/*']
```

Comment:
ditto: this is also mutable, maybe just do:

REGION_TAG_GREP_ARGS = (
  'grep',
  '-hr',
  'START',
  '.',
  '--include=*.py',
  ....etc
)

Reply:
Done.